x-pack/test/security_solution_cypress/cypress/tasks/timeline.ts

```diff
@@ -344,6 +346,29 @@ export const expandFirstTimelineEventDetails = () => {
   cy.get(TOGGLE_TIMELINE_EXPAND_EVENT).first().click({ force: true });
 };
 
+export const saveTimeline = () => {
+  cy.get(TIMELINE_SAVE_MODAL_OPEN_BUTTON).first().click();
+
+  cy.get('[data-test-subj="save-timeline-modal"]').within(() => {
```

Comment:
NIT: Please, wrap all the selectors in meaningful screen constants.

Reply:
Good point! Did it here: https://github.com/elastic/kibana/pull/169239/commits/320fd676e41fb962b8b488c1e51f777af85abdec